docs/sql-programming-guide.md

```diff
@@ -1087,6 +1087,13 @@ the following case-sensitive options:
   </tr>
 
   <tr>
+     <td><code>maxConnections</code></td>
+     <td>
+       The number of JDBC connections, which specifies the maximum number of simultaneous JDBC connections that are allowed. This option applies only to writing with <code>coalesce<code/> operation if needed. It defaults to the number of partitions of RDD.
```

Comment:
Pardon me for coming in with an additional comment here, but I would suggest yet another version of this text for more clarity, at least in my opinion:

If set, limits the maximum number of JDBC connections that can be used. Only applies when writing. It works by limiting the operation's parallelism, which depends on the input's partition count. If its partition count exceeds this limit, the operation will `coalesce` the input to fewer partitions before writing.